client/blocks/app-promo/style.scss

```diff
@@ -19,8 +19,8 @@
 .app-promo .app-promo__dismiss {
 	position: absolute;
 	right: 17px;
-	top: 12px;
```

Comment:
Right now, there's a difference between production and this PR as to where the dismiss button appears (vertically) - it's a bit subtle:

![](https://cldup.com/iCw6VBpJss.png) ![](https://cldup.com/lC32Gpy-NC.png)

Bringing this line back but changing it to `top: 3px` does the job for me.